src/main/resources/org/sagebionetworks/web/client/widget/header/HeaderViewImpl.ui.xml

```diff
@@ -9,8 +9,8 @@
 		type='org.sagebionetworks.web.client.SageImageBundle' />
 
 	<bh:Div addStyleNames="header">
-		<b:Alert ui:field="portalAlert" visible="false"
-			addStyleNames="fullWidthAlert flexcontainer-row background-color-white" type="INFO" height="50px"
+		<bh:Div ui:field="portalAlert" visible="false"
+			addStyleNames="padding-15 flexcontainer-row background-color-white"
```

Comment:
Is this no longer full-width at the top of the page, or is this maintaining its style some other way? Just double-checking because there is no screenshot

Reply:
Oh yeah, should have included a screenshot of this too.  It's a special case, the portal back bar.  It's no longer an Alert (because the new style conflicts with the portal back bar design):  
<img width="1079" alt="Screen Shot 2021-03-02 at 1 53 27 PM" src="https://user-images.githubusercontent.com/1864447/109720336-c9570e00-7b5e-11eb-90eb-07108ae30269.png">